scripts/manage_flavours.py

```diff
@@ -36,6 +34,14 @@ def parse_args():
         required=False,
         help="Updates flavor data in DB",
     )
+    parser.add_argument(
+        "-p",
+        "--prune",
+        action="store_true",
+        default=False,
+        required=False,
+        help="Prune flavours on database but not in config",
```

Comment:
Maybe, "Prune flavours on database that are not in yaml config"?

Reply:
Indeed.